### PR TITLE
[4.0] Set default workflow for protected workflows

### PR DESCRIPTION
**Pull request by bembelimen**
Pull Request for Issue #23465 .

### Summary of Changes
Enable changing the default workflow again.

### Testing Instructions
See the issue for testing instructions.

### CODE REVIEW

File: administrator/components/com_workflow/tmpl/workflows/default.php [hunk 1]
```diff
@@ -133,7 +133,7 @@
 								</td>
 								<td class="text-center">
 									<div class="btn-group">
-										<?php echo HTMLHelper::_('jgrid.published', $item->published, $i, 'workflows.', $canChange); ?>
+										<?php echo HTMLHelper::_('jgrid.published', $item->published, $i, 'workflows.', $canChange  && !$isCore); ?>
```
Comment (Quy) [inline administrator/components/com_workflow/tmpl/workflows/default.php]:
```suggestion
										<?php echo HTMLHelper::_('jgrid.published', $item->published, $i, 'workflows.', $canChange && !$isCore); ?>
```

**Comment by brianteeman** [comment]
You can now switch the default by clicking on the icon - so that is correct 
BUT as a consequence I see that the "Joomla Default" workflow now appears to be editable although it is not - I agree it should not be editable but then the pencil icon should not be displayed

chrome_2019-01-26_14-13-21

**Comment by bembelimen** [comment]
Thanks @brianteeman fixed it.

**Comment by brianteeman** [comment]
I can confirm the pencil issue is now resolved. 

But I now notice that you cannot unpublish that workflow even if it no longer the default. Is that intended?

**Comment by bembelimen** [comment]
> But I now notice that you cannot unpublish that workflow even if it no longer the default. Is that intended?

From my side it was intended, yes, but I have no feelings about it...it was a wish to have the default workflow unchangeable...

**Comment by brianteeman** [comment]
OK I will mark it as a successful test then

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e7602ee1cd3f0fca6cb33a06929fec2cdd2e1dba<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23607">issues.joomla.org/tracker/joomla-cms/23607</a>.</sub>

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on e7602ee1cd3f0fca6cb33a06929fec2cdd2e1dba<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23607">issues.joomla.org/tracker/joomla-cms/23607</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23607">issues.joomla.org/tracker/joomla-cms/23607</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**